### ISSUE TITLE
Modal body color combinations fail color contrast for light theme

### ISSUE THREAD

**Issue by ip69719** [ip69719/ci-portfolio-two#9]
Modal body color combinations fail color contrast. This needs to be resolved in conjunction with modal background color for light theme issue.

<img width="451" alt="image" src="https://user-images.githubusercontent.com/83339812/200831351-8455af37-3f69-41ee-9980-717d935b53e5.png">

**Comment by ip69719** [comment]
Replaced values for modal background color with CSS variables and set the background color for light theme to white. Color contrast test pass.
